e2e/testdata/fn-render/invalid-subpkg/db/Kptfile

```diff
@@ -0,0 +1,14 @@
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: db
+pipeline:
+  mutators:
+  - image: gcr.io/kpt-fn/starlark:unstable
+configPath: statefulset-filter.yaml
```

Comment:
what is this testcase testing? `invalid-subkg` is not very descriptive. Is it this malformed Kptfile?

Reply:
Yes, subpkg has invalid kptfile. Renamed it to `subpkg-has-invalid-kptfile`.